src/main/scala/chisel3/stage/ChiselAnnotations.scala

```diff
@@ -93,12 +93,36 @@ case object WarningsAsErrorsAnnotation
     new ShellOption[Unit](
       longOption = "warnings-as-errors",
       toAnnotationSeq = _ => Seq(WarningsAsErrorsAnnotation),
-      helpText = "Treat warnings as errors"
+      helpText = "Treat warnings as errors",
+      helpValueName = Some("<file>")
```

Comment:
Nit: unrelated change.

Reply:
Lol I put that in the wrong place, that was supposed to be on `SourceRootAnnotation`, will fix